cognitedata/reveal
# fix(react-components): search hook fix for instances with multiple connections to other 3d mapped instances

#### Type of change

![Bug](https://img.shields.io/badge/Type-Bug-red) 

#### Jira ticket :blue_book:


https://cognitedata.atlassian.net/browse/BND3D-3522

## Description :pencil:

Fixes an issue when "second-level" mapped 3d instances were not always appearing in search results.

## How has this been tested? :mag:


In FDX and searchHooks story.


## Test instructions :information_source:


1. Go to searchHooks story 
2. Search for "work-order"
3. Observe an item with the type "WorkOrderMultiple" in the search results. It was not there before the fix.

## Checklist :ballot_box_with_check:
